src/db/db.js

```diff
@@ -79,19 +79,56 @@ class Db {
   }
 
   /**
-   * Decrypts the encrypted packed data by the exportLocalStorage function, and saves it into localStorage.
-   *
+   * Decrypts the encrypted packed data and saves it into localStorage
    * @param  {Object} kc - KeyContainer
-   * @param  {String} encryptedDB
+   * @param  {String} Database encrypted
    */
-  importLocalStorage(kc, encryptedDB) {
-    const dbExpStr = kcUtils.decrypt(kc.encryptionKey, encryptedDB);
+  importLocalStorage(kc, dbEncrypted) {
+    const dbExpStr = kcUtils.decrypt(kc.encryptionKey, dbEncrypted);
     const dbExp = JSON.parse(dbExpStr);
 
     Object.keys(dbExp).forEach((key) => {
       localStorage.setItem(key, dbExp[key]);
     });
   }
+
+  /**
+   * Gets all the localStorage data and packs it into an encrpyted string
+   * @param  {Object} kc - KeyContainer
+   * @returns {String} - Encrypted packed data
+   */
+  exportWallet(kc) {
+    if (!kc.encryptionKey) {
+      // KeyContainer not unlocked
+      console.error('Error: KeyContainer not unlocked');
+      return undefined;
+    }
+    try {
+      const lsEncrypted = JSON.stringify(localStorage);
```

Comment:
this `lsEncrypted` is not encrypted, right?

Reply:
Totally true :+1: 
Label fixed :smile_cat: